pkg/agent/controller/networkpolicy/l7engine/reconciler.go

```diff
@@ -261,18 +266,27 @@ func convertProtocolTLS(tls *v1beta.TLSProtocol) string {
 }
 
 func (r *Reconciler) StartSuricataOnce() {
-	r.once.Do(func() {
+	r.onceStartSuricata.Do(func() {
 		r.startSuricata()
 	})
 }
 
+func (r *Reconciler) ensureL7NPFlows() {
```

Comment:
Probably rename it 'installL7NPFlowsOnce'

Reply:
Done